test/extensions/filters/http/proto_api_scrubber/scrubbing_util_lib/field_checker_test.cc

```diff
@@ -35,6 +36,8 @@ class MockProtoApiScrubberFilterConfig : public ProtoApiScrubberFilterConfig {
 namespace {
 
 inline constexpr const char kApiKeysDescriptorRelativePath[] = "test/proto/apikeys.descriptor";
+inline constexpr char kRemoveFieldActionType[] =
```

Comment:
seems a bit strange that it is defined here, and not somewhere in the codebase or inferred from the actual type.